test/stat_tracker_test.rb

```diff
@@ -197,6 +197,37 @@ def test_it_can_return_games_won_per_team_for_a_season
   def test_it_can_return_total_games_per_team_for_a_season
     assert_equal ({"3"=>5, "6"=>9, "5"=>4, "17"=>1, "16"=>1}), @@stat_tracker.total_games_per_team_for("20122013")
   end
+
+  def test_it_can_return_all_coaches
+    expected = ["John Tortorella", "Claude Julien", "Dan Bylsma", "Mike Babcock", "Joel Quenneville", "Jon Cooper", "Mike Johnston", "Alain Vigneault", "Peter DeBoer", "Gerard Gallant", "Randy Carlyle"]
+   assert_equal expected, @@stat_tracker.all_coaches
+  end
+
+  def test_it_can_get_team_games_by_coach
+    assert_instance_of Hash, @@stat_tracker.create_hash_with_team_games_by_coach("20122013")
+    assert_equal 11, @@stat_tracker.create_hash_with_team_games_by_coach("20122013").length
+  end
+
+  def test_it_can_get_games_by_coach_and_season
+    assert_instance_of Array, @@stat_tracker.game_teams_that_season_by_coach("Mike Babcock", 20122013)
+    assert_equal 9, @@stat_tracker.game_teams_that_season_by_coach("Claude Julien", 20122013).length
+  end
+
+  def test_number_of_games_by_coach
```

Comment:
Maybe for these last three methods we could add `_for_season` or something so that it makes it clear we're passing in an arg here.

Reply:
Second this!